### ISSUE TITLE
[Issue]: Webapp stuck on start-up, appears to be loading models?

### ISSUE THREAD

**Issue by Parasiterex** [vladmandic/automatic#535]
### Issue Description

After launching the app via webui.bat it appears as if the models keep loading, in this mode I am not able to generate images.
I have pasted the cmdline output and a screenshot of the ui
It was also tested without a model installed, and then installing the default model directly from the webui.bat

https://imgur.com/a/enaLeUH

The version was installed on 26/4/2023

venv "C:\Users\Parasiterex\automatic\venv\Scripts\Python.exe"
22:04:59-000485 INFO     Python 3.10.11 on Windows
22:05:00-001781 INFO     Version: 93b0de7e Wed Apr 26 09:02:32 2023 -0400
22:05:00-360037 INFO     Latest published version: e83708284a6103c15532e46578a996b5a3f35869 2023-04-26T19:54:32Z
22:05:00-364039 INFO     Setting environment tuning
22:05:00-369728 INFO     nVidia toolkit detected
22:05:04-627131 INFO     Torch 2.0.0+cu118
22:05:04-650143 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
22:05:04-654143 INFO     Torch detected GPU: NVIDIA GeForce RTX 3060 VRAM 12288 Arch (8, 6) Cores 28
22:05:04-657143 INFO     Verifying requirements
22:05:04-827431 INFO     No changes detected: Quick launch active
22:05:04-842418 INFO     Running extension preloading
22:05:04-844417 INFO     Server arguments: []
Unloading xFormers
Available models: C:\Users\Parasiterex\automatic\models\Stable-diffusion 1
ControlNet v1.1.112
ControlNet v1.1.112
Loading theme: black-orange
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Initializing middleware
Checkpoint model.ckpt not found; loading fallback v1-5-pruned-emaonly.safetensors [6ce0161689]
Loading weights: C:\Users\Parasiterex\automatic\models\Stable-diffusion\v1-5-pruned-emaonly.safetensors ------ 0.0/4… -:--:--
                                                                                                         GB
Creating model from config: C:\Users\Parasitrex\automatic\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying scaled dot product cross attention optimization
Embeddings loaded:  (0)
Model loaded in 20.3s (load=1.0s create=0.6s apply=10.1s vae=0.9s move=1.2s embeddings=6.5s)
Startup time: 46.0s (torch=7.2s gradio=2.9s libraries=2.2s codeformer=0.3s gfpgan=0.1s scripts=5.0s ui=7.3s start=0.2s
scripts app_started_callback=0.1s checkpoint=20.6s)

Setup.log

2023-04-26 22:23:25,699 | INFO | C:\Users\Parasiterex\automatic\setup.py | Python 3.10.11 on Windows
2023-04-26 22:23:25,841 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Git 2.39.2.windows.1
2023-04-26 22:23:26,693 | INFO | C:\Users\Parasiterex\automatic\setup.py | Version: 93b0de7e Wed Apr 26 09:02:32 2023 -0400
2023-04-26 22:23:27,024 | INFO | C:\Users\Parasiterex\automatic\setup.py | Latest published version: def7a02d82f4e7456177f3dbdf697dc7d02a3214 2023-04-26T20:08:56Z
2023-04-26 22:23:27,028 | INFO | C:\Users\Parasiterex\automatic\setup.py | Setting environment tuning
2023-04-26 22:23:27,033 | INFO | C:\Users\Parasiterex\automatic\setup.py | nVidia toolkit detected
2023-04-26 22:23:27,035 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: torch 2.0.0+cu118
2023-04-26 22:23:27,036 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: torchvision 0.15.1+cu118
2023-04-26 22:23:27,036 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: torchaudio 2.0.1+cu118
2023-04-26 22:23:29,726 | INFO | C:\Users\Parasiterex\automatic\setup.py | Torch 2.0.0+cu118
2023-04-26 22:23:29,764 | INFO | C:\Users\Parasiterex\automatic\setup.py | Torch backend: nVidia CUDA 11.8 cuDNN 8700
2023-04-26 22:23:29,768 | INFO | C:\Users\Parasiterex\automatic\setup.py | Torch detected GPU: NVIDIA GeForce RTX 3060 VRAM 12288 Arch (8, 6) Cores 28
2023-04-26 22:23:29,771 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: tensorflow 2.12.0
2023-04-26 22:23:29,771 | INFO | C:\Users\Parasiterex\automatic\setup.py | Verifying requirements
2023-04-26 22:23:29,774 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: addict 2.4.0
2023-04-26 22:23:29,774 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: aenum 3.1.12
2023-04-26 22:23:29,775 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: aiohttp 3.8.4
2023-04-26 22:23:29,775 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: anyio 3.6.2
2023-04-26 22:23:29,775 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: appdirs 1.4.4
2023-04-26 22:23:29,776 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: astunparse 1.6.3
2023-04-26 22:23:29,776 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: basicsr 1.4.2
2023-04-26 22:23:29,777 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: bitsandbytes 0.38.1
2023-04-26 22:23:29,777 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: blendmodes 2022
2023-04-26 22:23:29,777 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: clean-fid 0.1.35
2023-04-26 22:23:29,777 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: easydev 0.12.1
2023-04-26 22:23:29,778 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: extcolors 1.0.0
2023-04-26 22:23:29,778 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: facexlib 0.3.0
2023-04-26 22:23:29,779 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: filetype 1.2.0
2023-04-26 22:23:29,779 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: font-roboto 0.0.1
2023-04-26 22:23:29,779 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: fonts 0.0.3
2023-04-26 22:23:29,780 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: future 0.18.3
2023-04-26 22:23:29,780 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: gdown 4.7.1
2023-04-26 22:23:29,780 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: gfpgan 1.3.8
2023-04-26 22:23:29,781 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: GitPython 3.1.31
2023-04-26 22:23:29,781 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: httpcore 0.17.0
2023-04-26 22:23:29,781 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: inflection 0.5.1
2023-04-26 22:23:29,782 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: jsonmerge 1.9.0
2023-04-26 22:23:29,782 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: kornia 0.6.12
2023-04-26 22:23:29,783 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: lark 1.1.5
2023-04-26 22:23:29,783 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: lmdb 1.4.1
2023-04-26 22:23:29,783 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: lpips 0.1.4
2023-04-26 22:23:29,783 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: numpy 1.23.5
2023-04-26 22:23:29,784 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: omegaconf 2.3.0
2023-04-26 22:23:29,785 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: open-clip-torch 2.17.2
2023-04-26 22:23:29,785 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: opencv-contrib-python 4.7.0.72
2023-04-26 22:23:29,786 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: piexif 1.1.3
2023-04-26 22:23:29,786 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: Pillow 9.5.0
2023-04-26 22:23:29,786 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: psutil 5.9.5
2023-04-26 22:23:29,787 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: pyyaml 6.0
2023-04-26 22:23:29,787 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: realesrgan 0.3.0
2023-04-26 22:23:29,787 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: requests 2.29.0
2023-04-26 22:23:29,788 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: resize-right 0.0.2
2023-04-26 22:23:29,788 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: rich 13.3.4
2023-04-26 22:23:29,788 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: safetensors 0.3.1
2023-04-26 22:23:29,789 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: scipy 1.10.1
2023-04-26 22:23:29,789 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: tb_nightly 2.13.0a20230426
2023-04-26 22:23:29,789 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: toml 0.10.2
2023-04-26 22:23:29,790 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: torch 2.0.0+cu118
2023-04-26 22:23:29,790 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: torchdiffeq 0.2.3
2023-04-26 22:23:29,791 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: torchsde 0.2.5
2023-04-26 22:23:29,791 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: torchvision 0.15.1+cu118
2023-04-26 22:23:29,791 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: tqdm 4.65.0
2023-04-26 22:23:29,792 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: voluptuous 0.13.1
2023-04-26 22:23:29,792 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: yapf 0.33.0
2023-04-26 22:23:29,792 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: scikit-image 0.20.0
2023-04-26 22:23:29,793 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: accelerate 0.18.0
2023-04-26 22:23:29,793 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: opencv-python 4.7.0.72
2023-04-26 22:23:29,793 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: diffusers 0.15.0
2023-04-26 22:23:29,794 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: einops 0.4.1
2023-04-26 22:23:29,794 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: gradio 3.23.0
2023-04-26 22:23:29,794 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: numexpr 2.8.4
2023-04-26 22:23:29,795 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: pandas 1.5.3
2023-04-26 22:23:29,795 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: protobuf 3.20.3
2023-04-26 22:23:29,795 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: pytorch_lightning 1.9.4
2023-04-26 22:23:29,796 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: transformers 4.26.1
2023-04-26 22:23:29,796 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: timm 0.6.13
2023-04-26 22:23:29,797 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: tomesd 0.1.2
2023-04-26 22:23:29,797 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: xformers 0.0.18
2023-04-26 22:23:29,925 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Repository update time: Wed Apr 26 15:02:32 2023
2023-04-26 22:23:29,925 | INFO | C:\Users\Parasiterex\automatic\setup.py | Running setup
2023-04-26 22:23:29,928 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Args: {'f': False, 'ui_settings_file': 'C:\\Users\\Parasiterex\\automatic\\config.json', 'ui_config_file': 'C:\\Users\\Parasiterex\\automatic\\ui-config.json', 'config': 'C:\\Users\\Parasiterex\\automatic\\configs\\v1-inference.yaml', 'theme': None, 'medvram': False, 'lowvram': False, 'lowram': False, 'ckpt': 'C:\\Users\\Parasiterex\\automatic\\model.ckpt', 'vae': None, 'data_dir': 'C:\\Users\\Parasiterex\\automatic', 'models_dir': 'models', 'allow_code': False, 'share': False, 'enable_insecure': False, 'use_cpu': [], 'listen': False, 'port': None, 'hide_ui_dir_config': False, 'freeze_settings': False, 'gradio_auth': None, 'gradio_auth_path': None, 'autolaunch': False, 'disable_console_progressbars': True, 'disable_safe_unpickle': True, 'api_auth': None, 'api_log': False, 'device_id': None, 'cors_origins': None, 'cors_regex': None, 'tls_keyfile': None, 'tls_certfile': None, 'server_name': None, 'no_hashing': False, 'no_download_sd_model': False, 'profile': False, 'disable_queue': False, 'debug': False, 'reset': False, 'upgrade': False, 'noupdate': False, 'skip_requirements': False, 'skip_extensions': False, 'skip_git': False, 'experimental': False}
2023-04-26 22:23:29,928 | INFO | C:\Users\Parasiterex\automatic\setup.py | Installing packages
2023-04-26 22:23:29,930 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Package version found: clip 1.0
2023-04-26 22:23:29,930 | INFO | C:\Users\Parasiterex\automatic\setup.py | Installing repositories
2023-04-26 22:23:30,500 | INFO | C:\Users\Parasiterex\automatic\setup.py | Installing submodules
2023-04-26 22:23:34,856 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Submodules list: ce584a0ff863de98233ee135dcc17f2fb44703c3 extensions-builtin/a1111-sd-webui-lycoris (heads/main)
 9e6bbd9b8931bbe869a8e28e7005b0e13c2efff0 extensions-builtin/clip-interrogator-ext (heads/main)
 f3d79a474b9795f07143eaf8104737a403b5fb52 extensions-builtin/multidiffusion-upscaler-for-automatic1111 (heads/main)
 5d3465c4b2d5dacfdd6caa38b71b23b4f88401c0 extensions-builtin/sd-dynamic-thresholding (heads/master)
 896b6aa034127e8064ae5fdd6370f351e0755475 extensions-builtin/sd-extension-aesthetic-scorer (heads/main)
 90663eb7450c3487b693cf20e76ec4d7edd78cd5 extensions-builtin/sd-extension-steps-animation (heads/main)
 cc86ce8887f041e88e67d336044190fd0296fd74 extensions-builtin/sd-extension-system-info (heads/main)
 93b0f9e1b7cc246165666b7b307bc8243db2c3f4 extensions-builtin/sd-webui-controlnet (heads/main)
 d19e28168268b0f2f50c8a5b7a4fa0a0d9b42b8c extensions-builtin/sd-webui-model-converter (heads/main)
 ffe0553c59e91067ebf1e4fc7ad85ca9c870bf57 extensions-builtin/seed_travel (heads/main)
 704e42c10d01e6c6965493ec956a82bb8fc2da51 extensions-builtin/stable-diffusion-webui-images-browser (heads/main)
 64821f04767b04b92ac157f3c02f28d269dba5d8 extensions-builtin/stable-diffusion-webui-rembg (heads/master)
+18f171d885d4c870bd1c0656f7247e9649df62fd modules/lora (v0.6.2-17-g18f171d)
 8b47a5349b59efac812664c5732c6e3947a6fec7 modules/lycoris (heads/main)
 12603bcdec55df780b18612d58b6d0dcd4c27f96 wiki (heads/master)
2023-04-26 22:23:37,299 | INFO | C:\Users\Parasiterex\automatic\setup.py | Updating submodules
2023-04-26 22:23:41,756 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/a1111-sd-webui-lycoris / main
2023-04-26 22:23:42,801 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/clip-interrogator-ext / main
2023-04-26 22:23:43,819 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/multidiffusion-upscaler-for-automatic1111 / main
2023-04-26 22:23:44,829 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/sd-dynamic-thresholding / master
2023-04-26 22:23:47,951 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/sd-extension-aesthetic-scorer / main
2023-04-26 22:23:49,022 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/sd-extension-steps-animation / main
2023-04-26 22:23:50,020 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/sd-extension-system-info / main
2023-04-26 22:23:51,038 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/sd-webui-controlnet / main
2023-04-26 22:23:52,046 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/sd-webui-model-converter / main
2023-04-26 22:23:53,054 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/seed_travel / main
2023-04-26 22:23:54,041 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/stable-diffusion-webui-images-browser / main
2023-04-26 22:23:55,067 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: extensions-builtin/stable-diffusion-webui-rembg / master
2023-04-26 22:23:56,079 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: modules/lora / main
2023-04-26 22:23:57,114 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: modules/lycoris / main
2023-04-26 22:23:58,133 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: wiki / master
2023-04-26 22:23:59,116 | INFO | C:\Users\Parasiterex\automatic\setup.py | Extensions enabled: ['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR', 'Lora', 'multidiffusion-upscaler-for-automatic1111', 'prompt-bracket-checker', 'ScuNET', 'sd-dynamic-thresholding', 'sd-extension-aesthetic-scorer', 'sd-extension-steps-animation', 'sd-extension-system-info', 'sd-webui-controlnet', 'sd-webui-model-converter', 'seed_travel', 'stable-diffusion-webui-images-browser', 'stable-diffusion-webui-rembg', 'SwinIR']
2023-04-26 22:23:59,236 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\a1111-sd-webui-lycoris / main
2023-04-26 22:24:00,253 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\clip-interrogator-ext / main
2023-04-26 22:24:01,162 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Running extension installer: C:\Users\Parasiterex\automatic\extensions-builtin\clip-interrogator-ext / C:\Users\Parasiterex\automatic\extensions-builtin\clip-interrogator-ext\install.py
2023-04-26 22:24:11,248 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\multidiffusion-upscaler-for-automatic1111 / main
2023-04-26 22:24:12,297 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\sd-dynamic-thresholding / master
2023-04-26 22:24:13,339 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\sd-extension-aesthetic-scorer / main
2023-04-26 22:24:14,360 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\sd-extension-steps-animation / main
2023-04-26 22:24:15,407 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\sd-extension-system-info / main
2023-04-26 22:24:16,308 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Running extension installer: C:\Users\Parasiterex\automatic\extensions-builtin\sd-extension-system-info / C:\Users\Parasiterex\automatic\extensions-builtin\sd-extension-system-info\install.py
2023-04-26 22:24:17,063 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\sd-webui-controlnet / main
2023-04-26 22:24:17,934 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Running extension installer: C:\Users\Parasiterex\automatic\extensions-builtin\sd-webui-controlnet / C:\Users\Parasiterex\automatic\extensions-builtin\sd-webui-controlnet\install.py
2023-04-26 22:24:18,682 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\sd-webui-model-converter / main
2023-04-26 22:24:19,910 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\seed_travel / main
2023-04-26 22:24:20,792 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Running extension installer: C:\Users\Parasiterex\automatic\extensions-builtin\seed_travel / C:\Users\Parasiterex\automatic\extensions-builtin\seed_travel\install.py
2023-04-26 22:24:21,538 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\stable-diffusion-webui-images-browser / main
2023-04-26 22:24:22,471 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Running extension installer: C:\Users\Parasiterex\automatic\extensions-builtin\stable-diffusion-webui-images-browser / C:\Users\Parasiterex\automatic\extensions-builtin\stable-diffusion-webui-images-browser\install.py
2023-04-26 22:24:23,242 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\extensions-builtin\stable-diffusion-webui-rembg / master
2023-04-26 22:24:24,169 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Running extension installer: C:\Users\Parasiterex\automatic\extensions-builtin\stable-diffusion-webui-rembg / C:\Users\Parasiterex\automatic\extensions-builtin\stable-diffusion-webui-rembg\install.py
2023-04-26 22:24:24,805 | INFO | C:\Users\Parasiterex\automatic\setup.py | Extensions enabled: []
2023-04-26 22:24:24,807 | INFO | C:\Users\Parasiterex\automatic\setup.py | Updating Wiki
2023-04-26 22:24:24,924 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\wiki / master
2023-04-26 22:24:26,003 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setting branch: C:\Users\Parasiterex\automatic\wiki\origin-wiki / master
2023-04-26 22:24:26,928 | DEBUG | C:\Users\Parasiterex\automatic\setup.py | Setup complete without errors: 1682540667
2023-04-26 22:24:26,935 | INFO | C:\Users\Parasiterex\automatic\setup.py | Running extension preloading
2023-04-26 22:24:26,945 | INFO | C:\Users\Parasiterex\automatic\launch.py | Server arguments: []
2023-04-26 22:24:26,947 | DEBUG | C:\Users\Parasiterex\automatic\launch.py | Starting WebUI

Running on 

OS Name	Microsoft Windows 10 Pro
Version	10.0.19045 Build 19045
Processor	AMD Ryzen 5 1600 Six-Core Processor, 3400 Mhz, 6 Core(s), 12 Logical Processor(s)
Videocard Nvidia3060 12gb

**Comment by vladmandic** [comment]
last messages on your console are:
> Model loaded in 20.3s (load=1.0s create=0.6s apply=10.1s vae=0.9s move=1.2s embeddings=6.5s)
> Startup time: 46.0s (torch=7.2s gradio=2.9s libraries=2.2s codeformer=0.3s gfpgan=0.1s scripts=5.0s ui=7.3s start=0.2s
scripts app_started_callback=0.1s checkpoint=20.6s)

I don't see what is the issue? It took 46sec to start (ok, thats a bit slow, but still fine) and now its ready???
if you have actual issue, post actual issue.

**Comment by Parasiterex** [comment]
Thanks for looking at this, it seems I was unclear, from the console it appears to be loaded, but in the webui (see imgur) the model loading part just keeps on loading, and images can not be generated. It will remain like that indefinitely, adding more and more seconds. I can run automatic1111 on this system, so I guess the hardware is OK.

**Comment by bbaaxx** [comment]
I am experiencing the same problem, the dropdown to select checkpoints seems to be loading the model, altough everything else seem to work fine except from the fact that I cannot change the checkpoint loaded at launch.
Tried on chrome and mozilla with the same results seems to be a Gradio Issue. (I am running the code on a remote linux host).

I noticed that, hitting the refresh checkpoint button a couple times and then refreshing the page will sometimes fix the problem.

I can reproduce the issue quite often so please do let me know if there are any logs/troubleshooting I can provide.

**Comment by electrashave** [comment]
> i've heard about this, but was never able to reproduce the behavior, so investigation is slow.

I can't say for certain that this is the EXACT same issue, but I was able to reproduce the infinite model loading
BUT only on mobile devices (android phone to be specific)
Steps I used to reproduce:
- fresh install of the webui
- do NOT run with `--disable-queue` (webui works as expected with this flag)
- run with `--listen`
- connect via mobile device on local network

I checked the dev tools console and it looks like it's getting stuck on `sd_model_callback` in ui.js for some reason as it keeps displaying "Loading XXXXX" over and over
Comparing the WebSocket messages sent when the issue is not present to when I can reproduce the issue, the model list is not being sent when you first load the webui which I suspect is the root cause

Screenshots showing what I mean:
https://imgur.com/a/KytHThv

That's as far as I've gotten, hope it helps!

**Comment by jaretburkett** [comment]
I have had this issue on Automatic1111. It happened when gradio was updated. It worked fine prior to that so I was stick on a commit before that happened. Just switched over to vladmatic and same issue. Everything crawled. Spent the past hour debugging. Tracked it down to a chrome plugin. 

**Ghostery**. Deactivate it, everything is fast and smooth, activate it, it crawls and never loads. Trusting the site isn't enough, it has to be deactivated completly. Not sure if this is the same situation for you, but I have been living with this hell for a while now and it finally works. Hope this helps.

**Comment by Parasiterex** [comment]
Hello all,

Thank you for the suggestions, removing **Ghostery** was in instant-fix.
Thanks for all the suggestions, and taking the time!

**Comment by electrashave** [comment]
> Hello all,
> 
> Thank you for the suggestions, removing **Ghostery** was in instant-fix. Thanks for all the suggestions, and taking the time!

Seems the issue I found is unrelated to yours then

**Comment by huskerd911** [comment]
> 

I can confirm that I see the same as @electrashave (including all of detail from DevTools) except I see it in Chrome on Windows 11, it doesn't need to be on mobile.

**Comment by huskerd911** [comment]
I had Chrome DevTools up waiting for the loading to get stuck. When it did, I paused  the debugger, and began to step out of the JavaScript; obviously it was stuck in a loop... what I did notice was that a Chrome Extension (FakeSpot) was appearing to be selected occasionally in the JavaScript context drop down on the Console tab. So I disabled the extension and I haven't had a problem since (and I could have this issue reliably yesterday).
